### DISCUSSION TITLE
Git initialization stuck for template Tier.run - Metered Billing Stripe Starter

### DISCUSSION THREAD

**Discussion by 0xw01f** [vercel/community#3426] [Help] [answered]
### Summary

Hello,
I recently wanted to use the Tier.run template (https://vercel.com/new/templates/next.js/tier).
I followed all the steps, and I tried three times (two times yesterday and once today :
- Create git repository
- Add storage
- Configure project (ENV variables)
But when it was supposed to deplay, it just got stuck and there was this error message :

An unexpected error occurred. Our team has already been notified and are working to resolve the issue, please try again shortly.

It's the next day and it still doesn't work.
I am pretty sure I did everything right in the configuration, I also followed those steps :
https://github.com/tierrun/tier-vercel-openai#environment-variables

To be honest I don't know if I should ask this here or if I should open an issue on the developper's repo.
I didn't find any another template that fits my needs that much.

Thank you.

### Example

https://gyazo.com/deb7382159f78693e0a7fd81bdce1ef5

### Steps to Reproduce

Try to create a new project using this template and follow the steps they recommed :
https://vercel.com/new/templates/next.js/tier

**Comment by swarnava** [comment]
Hi,

I just tried to deploy the app and couldn't reproduce the issue. Could you share a temporary dummy variable and share us a screencast?

**Reply by 0xw01f** [reply]
Hi, Thank you very much for your will to help.
I just resolved te issue by myself, I checked the console and it said that my email would be published. So i deactivated the email protection on my github account and it is cloning the repo now.

**Comment by 0xw01f** [comment, marked as answer]
I checked the console and I saw that the issue was due to my github account's email protection.
<img width="443" alt="image" src="https://github.com/vercel/community/assets/76235339/78391ea1-7678-47bf-891c-acd700020b95">
